pkg/proc/core/core.go

```diff
@@ -269,6 +270,13 @@ func (p *process) Checkpoints() ([]proc.Checkpoint, error) { return nil, nil }
 // ClearCheckpoint clears a checkpoint, but will only return an error for core files.
 func (p *process) ClearCheckpoint(int) error { return errors.New("checkpoint not found") }
 
+func (p *process) SupportsBPF() bool {
+	return false
+}
+
+func (dbp *process) SetUProbe(fnName string, args []ebpf.UProbeArgMap) {
```

Comment:
`panic("not implemented")` just in case?

Reply:
Done.